src/models/Login.ts

```diff
@@ -0,0 +1,39 @@
+export enum LoginFlowsType {
+    Password = "m.login.password",
+    Token = "m.login.token",
+    SSO = "m.login.sso",
+    EmailIdentity = "m.login.email.identity",
+    MSISDN = "m.login.msisdn",
```

Comment:
Pretty sure this is wrong in the spec

Reply:
I mean I'm pretty sure the spec is wrong: I think this is meant to be suffixed with `.identity`. Will have to double check at all the layers though...